src/test/java/seedu/contax/model/person/PersonTest.java

```diff
@@ -50,6 +50,15 @@ public void isSamePerson() {
         assertFalse(BOB.isSamePerson(editedBob));
     }
 
+    // @Test
+    // public void updateTags() {
+    //     Person modifiedAlice = ALICE.updateTags(new HashSet<>());
+    //
+    //     Person expectedAlice = new PersonBuilder(ALICE).withTags().build();
+    //
+    //     assertEquals(modifiedAlice, expectedAlice);
+    // }
```

Comment:
was this meant to be commented out

Reply:
Yes. This function was replaced with `Person#withTag` and `Person#withoutTag`.

Fixed in a588344